packages/css-library/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/css-library",
-  "version": "0.21.2",
+  "version": "0.22.0",
```

Comment:
We'll just increment this when we create the next compontent-library release so you can revert it.